packages/ag-charts-website/src/content/docs/charts-range-bar-series/index.mdoc

```diff
@@ -22,6 +22,69 @@ series: [
 
 {% chartExampleRunner title="Basic Range Column" name="basic-range-column" type="generated" options="{ \"enterprise\": true }" /%}
 
+## Range Column Labels
+
+It's possible to add labels to the range columns, by adding the following to the series config:
+
+```js
+label: {
+}
+```
+
+By default, a label will be shown for the `yHighKey` value and `yLowKey` value of each column.
+
+In the snippet below, a label formatter is used to return the integer part of the data value to display as the column labels.
+
+-   The params object passed into the formatter includes an `itemId` which can be used to identify whether the value is a `high` or a `low` value:
+
+```js
+label: {
+    formatter: ({ itemId, value }) => {
+        return `${itemId === 'low' ? 'L' : 'H'}: ${value.toFixed(0)}`;
+    };
+}
+```
+
+The formatter above creates labels as seen in the provided example:
+{% chartExampleRunner title="Range Column Label" name="range-column-labels" type="generated" options="{ \"enterprise\": true }" /%}
+
+## Multiple Range Column Series
+
+It is possible to combine multiple range column series into a single chart.
+
+Note that in the example below we:
+
+-   Set the category axis `groupPaddingInner` to `0` to close the gap between the two series
+-   Use `yName` to control the text that the legend displays
+-   Use `yLowName` and `yHighName` to control the text that the tooltip displays
+
+{% chartExampleRunner title="Multiple Range Column Series" name="multiple-range-columns" type="generated" options="{ \"enterprise\": true }" /%}
+
+{% note %}
+It's not possible to stack multiple range series.
```

Comment:
Seems a bit odd to call this out specifically?

Reply:
Removed